src/components/display/fewlines/AccountOverview/AccountOverview.tsx

```diff
@@ -2,19 +2,14 @@ import Link from "next/link";
 import React from "react";
 import styled from "styled-components";
 
-import { Container } from "../Container";
-import { H1 } from "../H1/H1";
-import { H2 } from "../H2/H2";
 import { LoginsIcon } from "../Icons/LoginsIcon/LoginsIcon";
 import { RightChevron } from "../Icons/RightChevron/RightChevron";
 import { NeutralLink } from "../NeutralLink/NeutralLink";
 import { ShadowBox } from "../ShadowBox/ShadowBox";
 
-const Account: React.FC = () => {
+const AccountOverview: React.FC = () => {
   return (
-    <Container>
-      <H1>Welcome to your account</H1>
-      <H2>First name last name</H2>
+    <>
```

Comment:
You don't need the fragment.

Reply:
I do, in the future we must add more `<ShadowBox />`